src/queries/all-open-prs-query.ts

```diff
@@ -1,19 +1,20 @@
 import { gql, TypedDocumentNode } from "@apollo/client/core";
 import { client } from "../graphql-client";
 import { GetAllOpenPRsAndCardIDs, GetAllOpenPRsAndCardIDsVariables } from "./schema/GetAllOpenPRsAndCardIDs";
+import { noNullish } from "../util/util";
 
 export const getAllOpenPRsAndCardIDsQuery: TypedDocumentNode<GetAllOpenPRsAndCardIDs, GetAllOpenPRsAndCardIDsVariables> = gql`
-query GetAllOpenPRsAndCardIDs($after: String) {
+query GetAllOpenPRsAndCardIDs($endCursor: String) {
```

Comment:
Maybe keep the variable name as `after` which would make it a smaller change, and it also makes more sense since it's the "after for this query" vs being the "end of the last one"?

Reply:
I see what you mean, but slightly prefer `endCursor` because it's the convention they've chosen for `gh cli --paginate`? https://cli.github.com/manual/gh_api#synopsis:~:text=For%20GraphQL%20requests%2C%20this%20requires%20that,set%20of%20fields%20from%20a%20collection.

Reply:
I see, so there's a motivation in keeping these copy-pastable...  Unfortunate that they made a bad choice.

I at least exercised my right to complain: cli/cli#2867 but they probably will not change it...